### PR TITLE
add help command

### PR DESCRIPTION
**Pull request by mteiste**

### CODE REVIEW

File: scripts/avn [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/env $PYTHON
+#! /usr/bin/env python3
```
Comment (ojarva) [inline scripts/avn]: I think officially dropping python support should not be part of this PR.

Comment (ojarva) [reply scripts/avn]: Also, when this is done, we should also drop tests for python2 from travis.

Comment (mteiste) [reply scripts/avn]: fixed
